internal/collector/deployment.go

```diff
@@ -227,6 +229,27 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_deployment_revision",
+			Type: metric.Gauge,
+			Help: "Sequence number representing a specific revision of the deployment controller represented by the annotaion 'deployment.kubernetes.io/revision'.",
+			GenerateFunc: wrapDeploymentFunc(func(d *v1.Deployment) *metric.Family {
+				if d.ObjectMeta.Annotations == nil {
+					return &metric.Family{}
+				}
+				revision, err := strconv.Atoi(d.ObjectMeta.Annotations["deployment.kubernetes.io/revision"])
```

Comment:
I would move `"deployment.kubernetes.io/revision" ` to a constant variable.

Reply:
Fixed typo and changed from Sequence number into Number, because the revision is not really a sequence.